tests/e2e/test_comprehensive.py

```diff
@@ -0,0 +1,610 @@
+"""Comprehensive end-to-end tests for ttmp32gme with real audio files."""
+
+import pytest
+import time
+import shutil
+import sqlite3
+from pathlib import Path
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.common.exceptions import TimeoutException, NoSuchElementException
+from mutagen.mp3 import MP3
+from mutagen.id3 import ID3, TIT2, TPE1, TALB, TDRC, APIC, TRCK
+from PIL import Image
+import io
+import subprocess
+
+
+# Fixtures directory
+FIXTURES_DIR = Path(__file__).parent.parent / 'fixtures'
+
+
+@pytest.fixture(scope="module")
+def test_audio_files():
```

Comment:
Implement this as a context manager that creates the files on enter and removes them on exit 

Then use with statements wherever the files are needed